eslint.config.mjs

```diff
@@ -71,7 +72,7 @@ const confusingBrowserGlobals = [
   'top',
 ];
 
-export default pluginTypescript.config(
+export default defineConfig(
```

Comment:
Sorry, I'm not sure I understand this line. Do we lose any typescript linting replacing it like this?